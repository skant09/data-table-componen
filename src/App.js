import React from 'react';
import { Provider } from 'react-redux'
import Table from './table';

import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">Datatable</header>
      <Table />
    </div>
  );
}

export default App;
