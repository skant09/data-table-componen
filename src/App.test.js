import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';


import App from './App';
import Table from './table';

it('renders without crashing', () => {
  shallow(<App />);
  const app = renderer.create(<App />).toJSON();
  expect(app).toMatchSnapshot();
});

it('renders with header and table', () => {
  const app = shallow(<App />);
  const header = <header className="App-header">Datatable</header>;
  const table = <Table />
  expect(app).toContainReact(header);
  expect(app).toContainReact(table);
});
