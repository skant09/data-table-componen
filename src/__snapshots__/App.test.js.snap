// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    Datatable
  </header>
  <div
    style={
      Object {
        "clear": "both",
        "fontSize": "10px",
        "margin": "0 auto",
        "maxHeight": "400px",
        "overflow": "scroll",
      }
    }
  >
    <table
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <thead
        style={
          Object {
            "background": "#ccc",
            "border": "none",
          }
        }
      >
        <tr>
          <th>
            #
          </th>
          <th>
            AlbumId
          </th>
          <th>
            id
          </th>
          <th>
            thumbnailUrl
          </th>
          <th>
            Title
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
